check_log_ng.py

```diff
@@ -696,6 +702,11 @@ def _check_log_multi(self, logfile_pattern, remove_seekfile=False, tag=''):
                 self._remove_old_seekfile_with_inode(logfile_pattern, tag)
             else:
                 self._remove_old_seekfile(logfile_pattern, tag)
+
+        # if set error_no_exist, return UNKNOWN.
+        if self.config['error_no_exist'] and log_exist == False:
```

Comment:
I think `== False` ( and `== True` ) is not good expression.
`and not log_exist` is better.

In this context, I think the `log_exist` variable is unnecessary, "`and not logfile_list`" is enough.
( Evaluation of "list" returns `false` if `len(list) == 0` )

Reply:
log_exist is removed. and use `len(logfile_list) == 0`